sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Azure.ResourceManager.MachineLearning.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>1.0.0-beta.1</Version>
+    <Description>
+      Azure management client SDK for Azure resource provider MachineLearningServices.
```

Comment:
We should update the description from MachineLearningServices to MachineLearning